bazel/repositories.bzl

```diff
@@ -112,7 +112,7 @@ def envoy_api_deps(skip_targets):
     native.git_repository(
         name = "envoy_api",
         remote = REPO_LOCATIONS["envoy_api"],
-        commit = "b9e89d760d4849812b43a0979aa756137563bf83",
+        commit = "11582c112f920233b358e89898316f2cda8dc59a",
```

Comment:
As mentioned in the main PR description, this is currently using my repo branch of envoy-api - so this would need merging before this PR.